furms-web-ui/src/main/java/io/imunity/furms/ui/utils/FormUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 Bixbit s.c. All rights reserved.
+ *  See LICENSE file for licensing information.
+ */
+
+package io.imunity.furms.ui.utils;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.data.binder.Binder;
+import io.imunity.furms.ui.views.site.settings.SiteSettingsDto;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static java.util.stream.Collectors.toList;
+
+public class FormUtils {
+
+	public static <T extends Component> Optional<T> findFormField(Binder<SiteSettingsDto> binder, Class<T> fieldType, String id) {
```

Comment:
This code is specific to sites currently. But to be dropped completely - see below.